Quick Bar dialog documentation (#15033)
home-assistant/home-assistant.github.io: source/_docs/tools/quick-open.md

---
title: "Quick Open Dialog"
description: "Dialog for quickly accessing entities or running common commands"
---

> You can access the "Quick Open Command Bar" dialog (similar to VSCode or Chrome Devtools) from anywhere in Home Assistant using hotkeys.

Reviewer: ```suggestion
You can access the "Quick Open Command Bar" dialog (similar to VS Code or Chrome Devtools) from anywhere in Home Assistant using hotkeys.
```

Author: Ha! Nice find.

<p class='img'>
  <img src='/images/docs/quick-open/quick-open-dialog.gif' alt='Quick open dialog'>
  Quick open bar for accessing entities and running commands
</p>

## Hotkeys

> | Mode  | Hotkey (Mac) | Hotkey (Win)

Reviewer: ```suggestion
| Mode  | Hotkey (Windows) | Hotkey (macOS)
```
These table headings were swapped causing it to show the wrong hotkeys underneath. And I think it's a little more professional to not abbreviate the OS.

| ------------- | ------------- |------------- |
| Entity Filter | Ctrl+P | ⌘+P |
| Command | Ctrl+Shift+P | ⌘+Shift+P |

## Switching Modes

<p class='img'>
  <img src='/images/docs/quick-open/quick-open-switch-modes.gif' alt='Quick open dialog'>
  Switch modes without losing filter text
</p>

When the dialog has already been launched, you can switch to other modes without losing your filter text.

| Current Mode | Action | Switches to Mode |
| ------------- | ------------- | ------------- |
| Entity Filter | Use [hotkey](#hotkeys) (or prepend ">" to input field) | Command
| Command | Use [hotkey](#hotkeys) (or remove ">" from input field) | Entity Filter

## Entity Filter Mode 

> *Hotkey: Ctrl+P (or ⌘+P)*

Reviewer: ```suggestion
*Hotkey: Ctrl+P (Windows) or ⌘+P (macOS)*
```

Similar to Developer Tools -> States view, but more lightweight, and accessible from anywhere in the application.

<p class='img'>
  <img src='/images/docs/quick-open/quick-open-entity-filter.gif' alt='Quick open dialog'>
  Filter for entities in quick open dialog's entity filter mode
</p>

Once launched, start typing your entity id (or ["bits and pieces" of your entity id](#search-by-bits-and-pieces-rather-than-complete-substring)) to get back a filtered list of entities.

Clicking on an entity will open the "More Info" dialog for that entity. This is helpful when, say, you are in the middle of writing an automation and need some quick insight about an entity but don't want to navigate away to Developer Tools.

## Command Mode

> *Hotkey: Ctrl+Shift+P (or ⌘+Shift+P)*

Reviewer: ```suggestion
*Hotkey: Ctrl+Shift+P (Windows) or ⌘+Shift+P (macOS)*
```
